README.md

```diff
@@ -23,6 +23,12 @@ You can find the full list of available tapioca clients [here](http://tapioca-wr
 
 To create new flavours, refer to [Building a wrapper](http://tapioca-wrapper.readthedocs.org/en/stable/buildingawrapper.html) in the documentation. There is also a [cookiecutter template](https://github.com/vintasoftware/cookiecutter-tapioca) to help bootstraping new API clients.
 
+## Contributing
+
+### Setup dev environment
+
+Tapioca-Wrapper is ready for development in with Dev Container. After downloading the code you just need to use Reopen in Container option and after building run `make setup-devcontainer` just once.
```

Comment:
Please put a link in "Dev Container" explaining what it is for people who never used it.